lenovo/yoga/6/13ALC6/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, pkgs, ...  }:
+
+{
+  imports = [
+    ../../../thinkpad/yoga.nix
+    ../../../../common/gpu/amd/default.nix
+  ];
+
+  boot.initrd.kernelModules = [ "ideapad_laptop" ];
+  hardware.opengl.extraPackages = with pkgs; [
+    vaapiVdpau
+    libvdpau-va-gl
+  ];
+
+  # Latest Kernel
+  boot.kernelPackages = lib.mkDefault pkgs.linuxPackages_latest;
```

We usually set a minimum kernel that supports the current hardware:

```suggestion
  boot.kernelPackages =  lib.mkIf (lib.versionOlder pkgs.linux.version "6.1") pkgs.linuxPackages_latest;
```

---

Since this model is 2018, do we need the latest kernel here at all? If it is just your personal preference to use the latest kernel than this would be better left your own configuration.

---

this model is 2021 and latest kernel needed to make wifi work